change/@ni-nimble-components-36d61393-7a39-455d-9100-e189f62e41aa.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Based on other comment if it's breaking lets do `major`, otherwise lets keep `minor`